features/netsocket/WiFiInterface.h

```diff
@@ -22,9 +22,8 @@
 #include "netsocket/NetworkInterface.h"
 #include "netsocket/WiFiAccessPoint.h"
 
-/** WiFiInterface class
+/** Common interface that is shared between WiFi devices.
```

Comment:
Wi-Fi

Reply:
Though this depends entirely on context. Chasing up.

Reply:
Instances of Wi-Fi that are not a class or variable should be styled as *Wi-Fi*, not *WiFi*.